### PR TITLE
Forward-port: 3.8.6 release notes

### PR DESCRIPTION
**Pull request by parkr**
This is the 3.8.6 release notes, but forward-ported to `master` (#7735).

It also updated the contributing documentation on the site after running `bundle exec rake site:generate`.

Once this is merged, I'll release 3.8.6.

### CODE REVIEW

File: docs/_docs/contributing.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ If you want to add your plugin to the [list of plugins](https://jekyllrb.com/doc
 
 ## Code Contributions
 
-Interested in submitting a pull request? Awesome. Read on. There's a few common gotchas that we'd love to help you avoid.
+Interesting in submitting a pull request? Awesome. Read on. There's a few common gotchas that we'd love to help you avoid.
```
Comment (DirtyF) [inline docs/_docs/contributing.md]: Interesting?

**Comment by ashmaroli** [comment]
> Once this is merged, I'll release 3.8.6.

The date on the release is `03-07-2019`. So I'll approve tomorrow..

**Comment by ashmaroli** [comment]
Oh wait! It says **`2018`**!!!

File: History.markdown [hunk 2]
```diff
@@ -305,6 +305,21 @@
 
   * Release v4.0.0.pre.alpha1 (#7574)
 
+## 3.8.6 / 2018-07-03
```
Comment (ashmaroli) [inline History.markdown]: :warning: Blast from the past!

**Comment by ashmaroli** [comment]
@parkr  What's the release date going to be..?

File: docs/_docs/contributing.md (hunk 1 above)
Comment (parkr) [reply docs/_docs/contributing.md]: This is copied/generated from the CONTRIBUTING.md file.

Comment (ashmaroli) [reply docs/_docs/contributing.md]: This was fixed in #7671

**Comment by ashmaroli** [comment]
## Requirements

The following are required before this can be merged:
- [ ] Review and merge #7737
- [ ] Update this branch to `master` following merge of #7737 and regenerate site
- [ ] Update Release Date on `3.8-stable` branch

**Comment by DirtyF** [comment]
@parkr Up to you :shipit:

**Pull request merged**